src/dashboard/src/components/filesystem_ajax/views.py

```diff
@@ -813,3 +815,25 @@ def download_fs(request):
     except ValueError:
         raise django.http.Http404
 
+def download(request, uuid):
```

Comment:
More specific name and/or docstring? Otherwise we have download_fs, download_ss and download with nothing clear to differentiate them.

It's probably out of scope to merge all the download_foos into one function, but it would be nice.